mmv1/third_party/terraform/website/docs/r/dataproc_cluster.html.markdown

```diff
@@ -14,7 +14,7 @@ Manages a Cloud Dataproc cluster resource within GCP.
 
 
 !> **Warning:** Due to limitations of the API, all arguments except
-`labels`,`cluster_config.worker_config.num_instances` and `cluster_config.preemptible_worker_config.num_instances` are non-updatable. Changing others will cause recreation of the
+`labels`,`cluster_config.worker_config.num_instances` and `cluster_config.preemptible_worker_config.num_instances` are non-updatable. Changing `cluster_config.worker_config.min_num_instances` will be ignored.Changing others will cause recreation of the
```

Comment:
worker_config.num_instances will not be ignored. worker_config.min_num_instances will.

The definition of worker_config.min_num_instances: The minimum number of primary worker instances to create. If min_num_instances is set, cluster creation will succeed if the number of primary workers created is at least equal to the min_num_instances number.

After cluster creation, updating min_num_instances makes no sense. Thus it will be ignored, as its ForceNew is false.  And highlight in the document.